GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -849,7 +849,7 @@ private void RefreshSelection()
             FillCommitInfo(selectedRevision);
 
             // If the revision's body has been updated then the grid needs to be refreshed to display it
-            if (selectedRevision is not null && selectedRevision.HasMultiLineMessage && oldBody != selectedRevision.Body)
+            if (AppSettings.ShowCommitBodyInRevisionGrid && selectedRevision is not null && selectedRevision.HasMultiLineMessage && oldBody != selectedRevision.Body)
```

Comment:
Is this correct when toggling the option?

Reply:
I also applied this from the original commit, but I think it is wrong.
Even if the option is disabled, the grid should be refreshed to display the multi-line indicator in case someone changed a commit message from single to multi-line, e.g. during amend.
This should probably be removed unless I'm wrong about the mechanics.

Reply:
I think it is correct. The comment above explains why.
`[Toggle]ShowCommitBodyInRevisionGrid()` calls `RevisionGridControl.Refresh()`.